contrail-ui/src/components/main/MainFrame.tsx

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from "react";
+import MainAppBar from "./main-app-bar/MainAppBar";
+import MainDrawer from "./main-drawer/MainDrawer";
+import MainView from "./main-view/MainView";
+
+class MainFrame extends Component {
+    public render() {
+        return (
+            <div>
```

Comment:
Does this need to be in div?

Reply:
Yes it does, but it's better to put in a React Fragment. I changed